dune-project

```diff
@@ -18,5 +18,19 @@
   (name memgraph)
   (depends (ocaml (>= 4.05.0)))
   (synopsis "A small library to inspect memory representation of ocaml values")
-  (description "Memgraph alows one to inspect an ocaml value and get a representation of its layout in memory, and helpers to dump such representation as dot files to easily print them as graphs")
+  (description "Memgraph allows one to inspect an ocaml value and get a representation of its layout in memory, and helpers to dump such representation as dot files to easily print them as graphs")
 )
+
+; Memgraph_kitty package
+(package
+  (name memgraph_kitty)
+  (depends
+     (ocaml (>= 4.05.0))
+     (ppx_blob (and :build (>= 0.7.0)))
+     (nanosvg (>= 0.1))
+     (nanosvg_text (>= 0.1))
+     (kittyimg (>= 0.1))
+     (stb_truetype (>= 0.7))
```

Comment:
you should probably add a dependency on `conf-graphviz` (which already handles the adequate depexts to make sure `dot` is available).

Reply:
there is! (the line below)

Reply:
oups, I guess I glossed over that because it didn't have version constraint, my bad, ^^